src/lib/journey/callbacks/text-output/text-output.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+  import type { ConfirmationCallback, TextOutputCallback } from '@forgerock/javascript-sdk';
+  import Centered from '$components/primitives/box/centered.svelte';
+  import Button from '$components/primitives/button/button.svelte';
+
+  export let callback: TextOutputCallback;
+  export let choice: ConfirmationCallback;
+
+  const options = choice.getOptions();
+  $: selected = choice.getDefaultOption(); // gets the index position
+
+  const message = callback.getMessage();
+
+  function setValue(index: number) {
+    selected = index;
+  }
+</script>
+
+<Centered>
+  <div>{message}</div>
+  {#each options as opt, index}
+    <Button
+      style={selected === index ? 'primary' : 'secondary'}
+      type="button"
+      width="auto"
+      onClick={() => setValue(index)}
+    >
+      {opt}
```

Comment:
Are these options localizable within ForgeRock? In other words, does the admin set the option in English, Spanish, French ... Or, is it just one value per option? If it's the former, we can just render it how you're doing it here.

But, if it's the latter (not localization in ForgeRock), then we need to construct a key out of the option, and use that as a lookup to a content string from the locale file.

Reply:
Looks like it is the former.
![Screen Shot 2022-09-26 at 1 51 25 PM](https://user-images.githubusercontent.com/18267769/192367239-35fee685-5ced-4dc2-baf4-ae02a316637e.png)